clj/build.boot

```diff
@@ -51,6 +62,16 @@
            (install)
            (target)))
 
+(deftask build-cljs
+  "build clojurescript"
+  []
+  (comp
+   (test-cljs)
+   (cljs :optimizations :advanced)))
+
+(deftask build []
+  (comp (build-cljs) (build-clj)))
```

Comment:
Unfortunately, the composition doesn't work in this order. `build-clj` is not invoked.  I didn't investigate why. Suggestion is to change to:
```clojure
(deftask build-clj
         "build clojure"
         []
         (comp
           (pom)
           (test)
           (aot :all true)
           (jar)
           (install)
           (target)))

(deftask build-cljs
         "build clojurescript"
         []
         (comp
           (test-cljs)
           (cljs :optimizations :advanced)))

(deftask build
         []
         "build full project"
         (comp
           (build-clj)
           (build-cljs)))
```
In which case both sub-tasks get built.